### ISSUE TITLE
Sharing a JIG - - must have a response when URL is copied

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#2154]
When you click the button 'Copy URL', there is no response, and I was left wondering whether anything had happened. Only opening up a new tab and pasting proved that something had been copied.
Definitely need some sort of status 'URL copied to clipboard' or something like that to indicate that it has been copied. @ChaykeeMor?

![image](https://user-images.githubusercontent.com/16290361/147937838-8dd88413-5d2d-4e8b-b1da-5de337d0ceb3.png)

**Comment by ChaykeeMor** [comment]
I think within Sharing Options it should be 'Copy JIG link' and when clicked should change to 'JIG link copied'

**Comment by MendyBerger** [comment]
After 3 seconds text should change back to 'Copy JIG link'

**Comment by mastapegs** [comment]
This still needs to have the text change back after 3 seconds. should be done today

**Comment by corinnewo** [comment]
@mastapegs I still can't see this in sandbox. Also, now the share popup is not closing when you close the JIG panel, or when you click outside it. Is this connected with what you were doing for copying the URL?
![image](https://user-images.githubusercontent.com/16290361/150094221-2c883679-709e-4cc3-ae4b-d9d81fd416f6.png)

**Comment by mastapegs** [comment]
@corinnewo 

The PR for this is ready-for-review: https://github.com/ji-devs/ji-cloud/pull/2275

After it's been reviewed and merged into Sandbox, it'll be ready. I'll ping you once this is done.

![image](https://user-images.githubusercontent.com/13839150/150108787-fedaa546-2853-4652-9748-72923d628777.png)

**Comment by corinnewo** [comment]
@mastapegs This has still not been merged! What is happening here?
